x/slashing/keeper_test.go

```diff
@@ -129,3 +129,36 @@ func TestHandleAbsentValidator(t *testing.T) {
 	validator, _ = sk.GetValidatorByPubKey(ctx, val)
 	require.Equal(t, sdk.Unbonded, validator.GetStatus())
 }
+
+func TestHandleNewValidator(t *testing.T) {
```

Comment:
Can we add a small description (couple sentences) explaining the intention of this test... looks cool, I'd like to better understand the scenario

Reply:
Added a few-line description to all tests in this file.